geometry_msgs/PoseArray ugv_frontiers
---
float32[] ugv_frontier_visibility
